scripts/python/golioth/pyproject.toml

```diff
@@ -0,0 +1,35 @@
+[build-system]
```

Comment:
Any pros/cons for keeping this package here vs in an external repo? I think it would be nice to put it in an external repo, but there might be reasons for keeping it here too.

This is a really great tool, so I want to make sure it's visible and available to external users.

Reply:
Initially this was supposed to be bunch of scripts in the repo, included by pytest scripts or alternatively called by used by providing full path (CLI). Including those scripts from pytest scripts would be hacky (due to `../../../` usage), so I decided to make an installable package out of it. This way there is the CLI tool available without providing full path and pytest scripts can just import the module just as other Python modules/packages.

The reason why NOT to put it in separate repo for now is due to maintenance costs. Keeping all in single repo allows to quickly adjust and break APIs. And as this only supports HTTP and self-hosted server, there are no concerns at all with breaking something. And the main use-case is testing as I said (either automatic or manual), so I didn't want to increase testing effort even more by putting the maintenance of "full fledged" package to our team.

Reply:
Makes sense. We can keep it in the repo.